release/release-notes/data-prepper.release-notes-1.4.0.md

```diff
@@ -0,0 +1,36 @@
+## 2022-05-13 Version 1.4.0
+
+---
+
+### Features
+* Support metrics ingestion ([#1154](https://github.com/opensearch-project/data-prepper/pull/1154), [#1271](https://github.com/opensearch-project/data-prepper/pull/1271))
+
+### Enhancements
+* Migrate Trace Analytics plugins to Event Model ([#1216](https://github.com/opensearch-project/data-prepper/pull/1216), [#1223](https://github.com/opensearch-project/data-prepper/pull/1223), [#1224](https://github.com/opensearch-project/data-prepper/pull/1224), [#1220](https://github.com/opensearch-project/data-prepper/pull/1220), [#1237](https://github.com/opensearch-project/data-prepper/pull/1237), [#1241](https://github.com/opensearch-project/data-prepper/pull/1241), [#1239](https://github.com/opensearch-project/data-prepper/pull/1241))
+
+### Infrastructure
+* Added needs-documentation label ([#1373](https://github.com/opensearch-project/data-prepper/pull/1373))
+* Upload and publish JUnit test reports for some tests ([#1336](https://github.com/opensearch-project/data-prepper/pull/1336))
+
+### Documentation
+* Performance tests documentation ([#1309](https://github.com/opensearch-project/data-prepper/pull/1309))
+* Added custom metrics in otel-trace-source ([#1246](https://github.com/opensearch-project/data-prepper/pull/1246))
+
+### Maintenance
+* Support for OpenSearch 2.0 (previously supported versions of OpenSearch and OpenDistro are still supported)
```

Comment:
Sorry for the confusion on this. I do believe that this support should be an Enhancement. It has a direct impact on end-users. But, the two sub-bullets are more low-level details.

Reply:
This should be in the Enhancements:

>  Support for OpenSearch 2.0 (previously supported versions of OpenSearch and OpenDistro are still supported)

Reply:
No problem, Thanks for the feedback David.